en/audio-system/audiosource.md

```diff
@@ -7,13 +7,15 @@
 |Property          | Description                                                    |
 
 |:--                 | :--                                                                     |
-|Clip                | The audio resource object to be played                                           |
+|**Clip**                | The audio resource object to be played                                           |
 
-|Loop                | Whether to loop                                                      |
+|**Loop**                | Whether to loop                                                      |
 
-|PlayOnAwake         | Whether to play audio automatically after the component is activated    |
-|Volume              | Audio volume, ranging from 0 to 1                                  |
+|**PlayOnAwake**         | Whether to play audio automatically after the component is activated    |
+|**Volume**              | Audio volume, ranging from 0 to 1                                  |
 
+For additional details, please refer to [AudioSource API](../../../api/en/classes/component_audio.audiosource.html).

```

Comment:
This sentence is repeated with the following one, so suggest deleting it.

Reply:
oh, yup. thank you for catching.

Reply:
@slackmoehrle This repeated sentence is still not deleted.